Mage/src/main/java/mage/abilities/keyword/SpreeAbility.java

```diff
@@ -0,0 +1,27 @@
+package mage.abilities.keyword;
+
+import mage.abilities.StaticAbility;
+import mage.cards.Card;
+import mage.constants.Zone;
+
+/**
+ * @author TheElk801
+ */
+public class SpreeAbility extends StaticAbility {
+
+    public SpreeAbility(Card card) {
+        super(Zone.ALL, null);
+        this.setRuleVisible(false);
+        card.getSpellAbility().getModes().setChooseText("Spree <i>(Choose one or more additional costs.)</i>");
+        card.getSpellAbility().getModes().setMaxModes(3);
```

Comment:
Must use any amount of activations, not 3.

Reply:
yeah I'll just change it to `Integer.MAX_VALUE`